src/main/java/util/OrderPet.java

```diff
@@ -0,0 +1,31 @@
+package util;
+
+import io.restassured.http.ContentType;
+import models.Order;
+import models.Pet;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+public class OrderPet {
+    String BASE_URL = "https://petstore.swagger.io/v2/store/order/";
+
+    public void createOrder(Order newOrder) {
+        given().
+                contentType(ContentType.JSON).
+                body(newOrder).
+        when().
+                post(BASE_URL).
```

Comment:
if BASE_URL was a constant this could become
`post(BASE_URL + /store/order/)` minor improvement, but your code is cleaner